infra/cifuzz/docker.py

```diff
@@ -79,7 +80,8 @@ def get_base_docker_run_args(workspace,
       'OUT': workspace.out
   }
   docker_args += get_docker_env_vars(env_mapping)
-  docker_container = utils.get_container_name()
+  docker_container = environment.get('CFL_CONTAINER_ID',
+                                     utils.get_container_name())
```

Comment:
Gitlab runner's docker does not have its right name in /etc/hostname...
So, I use `export CFL_CONTAINER_ID=`cut -c9- < /proc/1/cpuset`
What do you think ?

Reply:
What happens if you don't set a docker container for this at all?

Reply:
Ping?

Reply:
Ping.

Reply:
Nevermind. I see your reply to this below.

Reply:
I guess this is because Gitlab CI uses somehow a different Docker thing (I do not know the right term : container/image) for each job step (`before_script`, `script`, ...) 

So hostname is `runner-adjcnmb--project-26-concurrent-0`
but the real name is `runner-adjcnmb--project-26-concurrent-0-585fb30ed19459ae-build-3`